spec/models/membership_spec.rb

```diff
@@ -1,18 +1,5 @@
 require "rails_helper"
 
-RSpec.describe Membership do
-  describe "associations" do
-    specify { expect(subject).to belong_to(:profile) }
-    specify { expect(subject).to belong_to(:organisation) }
-  end
-
-  describe "validations" do
-    specify { expect(subject).to validate_presence_of(:profile) }
-    specify { expect(subject).to validate_presence_of(:organisation) }
-
-    specify do
-      expect(subject).to validate_uniqueness_of(:profile_id).
-        scoped_to(:organisation_id)
-    end
-  end
+RSpec.describe Membership, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I think it would be good to add some specs here, considering we're using a the ActiveRecord + Postgres magic for the roles and applications.